br/tests/lightning_reload_cert/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright 2022 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+cp "$TEST_DIR/certs/lightning.pem" "$TEST_DIR/certs/lightning-valid.pem"
+trap 'mv "$TEST_DIR/certs/lightning-valid.pem" "$TEST_DIR/certs/lightning.pem"' EXIT
+
+# shellcheck disable=SC2089
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/lightning/SetCertExpiredSoon=return(\"$TEST_DIR/certs/ca.key\")"
+export GO_FAILPOINTS="${GO_FAILPOINTS};github.com/pingcap/tidb/br/pkg/lightning/restore/SlowDownWriteRows=sleep(15000)"
+
+run_lightning --backend='local'
```

Comment:
we should check the cert do expire(by log or something) and then success.

Reply:
emm,  lightning doesn't report any cert expired log, just connection close error.

Reply:
will tidb backend report this error?

Reply:
Ok, I will add some check later.

Reply:
I add a negative test for local backend in 82272b2d0d5a98842ce343f7cb60f4c5b4358ef8.

I found tidb backend doesn't report this error, because tls expiration check doesn't affect existing connection.